File: app/src/main/java/com/jormy/nin/NINView.java
package com.jormy.nin;

@Api
public class NINView {
    @Api
    public static float getDevicePPI() {
        return com.lurebat.keyboard71.NINView.Companion.getDevicePPI();

    }
    @Api
    public static float getDevicePortraitWidth() {
        return com.lurebat.keyboard71.NINView.Companion.getDevicePortraitWidth();
    }

    @Api
    public static void adjustWantedScaling(float scaling) {
        com.lurebat.keyboard71.NINView.Companion.adjustWantedScaling(scaling);
    }

    @Api
    public static void onRoenSignalDirty() {
        com.lurebat.keyboard71.NINView.Companion.onRoenSignalDirty();
    }

    @Api
    public static void onRoenFrozennessChange(boolean truth) {
        com.lurebat.keyboard71.NINView.Companion.onRoenFrozennessChange(truth);
    }

    @Api
    public static void adjustKeyboardDimensions(float wantedRoenHeight, boolean fullscreen) {
        com.lurebat.keyboard71.NINView.Companion.adjustKeyboardDimensions(wantedRoenHeight, fullscreen);
    }
}

File: app/src/main/java/com/jormy/nin/NINView.kt
package com.jormy.nin

@Api
class NINView {
    companion object {
        @Api
        @JvmStatic
        fun getDevicePPI(): Float {
            return com.lurebat.keyboard71.NINView.devicePPI

        }
        @Api
        @JvmStatic
        fun getDevicePortraitWidth(): Float {
            return com.lurebat.keyboard71.NINView.devicePortraitWidth
        }

        @Api
        @JvmStatic
        fun adjustWantedScaling(scaling: Float) {
            return com.lurebat.keyboard71.NINView.adjustWantedScaling(scaling)
        }

        @Api
        @JvmStatic
        fun onRoenSignalDirty() {
            return com.lurebat.keyboard71.NINView.onRoenSignalDirty()
        }

        @Api
        @JvmStatic
        fun onRoenFrozennessChange(truth: Boolean) {
            return com.lurebat.keyboard71.NINView.onRoenFrozennessChange(truth)
        }

        @Api
        @JvmStatic
        fun adjustKeyboardDimensions(wantedRoenHeight: Float, fullscreen: Boolean) {
            return com.lurebat.keyboard71.NINView.adjustKeyboardDimensions(wantedRoenHeight, fullscreen)
        }
    }
}
